cns/logger/cnslogger_windows.go

```diff
@@ -0,0 +1,29 @@
+package logger
+
+import (
+	"github.com/Azure/azure-container-networking/zapetw"
+	"github.com/pkg/errors"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	// LogPath is the path where log files are stored.
+	LogPath         = ""
```

Comment:
where is this used? why is in the platform-specific files?

Reply:
It is not used anywhere. Removing it.